etc/c/curl.d

```diff
@@ -879,8 +881,10 @@ enum CurlOption {
   /** The CApath or CAfile used to validate the peer certificate
      this option is used only if SSL_VERIFYPEER is true */
   cainfo = 10065,
-  /** 66 = OBSOLETE */
-  /** 67 = OBSOLETE */
+  /** $(UL */
```

Comment:
Doesn't look like it: http://dtest.thecybershadow.net/artifact/website-fa397df5041e797d4ecdefd0231aa6ae97f9ea34-b29971837d285bfc1b93b300157617fc/web/library-prerelease/etc/c/curl/curl_option.maxredirs.html